styles/component-style/input.css

```diff
@@ -0,0 +1,45 @@
+.input-container,input {
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+    background-color: var(--background-color);
+    padding:0.5rem;
+}
+
+.name-input, .password-input {
+    width:30rem;
+    background-color:white;
+}
+
+.name-input-success{
+    border: solid 2px green;
+}
+
+.password-input-error{
+    border: solid 2px red;
+}
+
+.input-validation {
+    font-size: small;
+}
+
+.name-input-validation {
+    color:green;
+}
+
+.password-input-validation {
+   color:red;
+}
+
+.input-btn-wrapper,.input-btn {
```

Comment:
You can use a utility class for this named as bricks-flex and pass it to the HTML file if you want to apply display:flex to that element